plugins/conference/services/ConferenceService.php

```diff
@@ -111,9 +113,17 @@ protected function findExistingConferenceRoom($entryId)
 		return null;
 	}
 
-	protected function findFreeServerNode()
+	protected function findFreeServerNode($tag, $partner)
 	{
-		$serverNodes = ServerNodePeer::retrieveActiveUnoccupiedServerNodesByType(ConferencePlugin::getCoreValue('serverNodeType',ConferenceServerNodeType::CONFERENCE_SERVER));
+		if (empty($tag))
+		{
+			$tag = $partner->getConfTagName();
```

Comment:
Better move the if check inside the getConfTagName i.e:
public function getConfTagName()
	{
		return $this->getFromCustomData( self::CUSTOMER_DATA_CONF_TAG_NAME, null ,kConf::get(self::CONFERENCE_SERVER_NODE_TAG_NAME));
	}